openreview/conference/templates/programchairWebfield.js

```diff
@@ -1224,14 +1277,30 @@ var displayPaperStatusTable = function() {
       );
     };
 
-    var filteredRows;
     if (searchText) {
-      filteredRows = _.orderBy(_.filter(rowData, filterFunc), sortOptions['Paper_Number'], 'asc');
-      matchingNoteIds = filteredRows.map(function(row) { return row.note.id; });
+      if (isQueryMode) {
+        var filterResult = Webfield.filterCollections(rowData, searchText.slice(1), filterOperators, propertiesAllowed, 'note.id');
+        filteredRows = filterResult.filteredRows;
+        queryIsInvalid = filterResult.queryIsInvalid;
+        if (queryIsInvalid) $(container + ' .form-search').addClass('invalid-value');
+      } else {
+        filteredRows = _.filter(rowData, filterFunc);
+      }
+    
+      filteredRows = _.orderBy(filteredRows, sortOptions['Paper_Number'], 'asc');
+      matchingNoteIds = filteredRows.map(function (row) {
+        return row.note.id;
+      });
+      order = 'asc'
     } else {
       filteredRows = rowData;
       matchingNoteIds = [];
     }
+    if (rowData.length !== filteredRows.length) {
+      conferenceStatusData.filteredNotes = filteredRows.map(function (p) {
```

Comment:
I would like to keep the status of `filteredRows` instead of `filteredNotes`, then the export button doesn't need to compute the rows again to export the stats. I know this is not related to this PR but I would reduce the duplicated code.